conf/common/requirements.txt

```diff
@@ -10,4 +10,4 @@ termcolor
 terminaltables
 tqdm
 yapf
-git+https://github.com/SymbiFlow/edalize.git@symbiflow#egg=edalize
+git+https://github.com/antmicro/edalize.git@symbiflow_update#egg=edalize
```

Comment:
Can you just push to the SymbiFlow repo?

Reply:
This PR is mainly to verify that the rebased version of edalize is working. As soon as it is verified to work we'll push to symbiflow and get the upstream PRs landed: https://github.com/olofk/edalize/pull/262, https://github.com/olofk/edalize/pull/261, https://github.com/olofk/edalize/pull/260